docs/identifierCase.md

```diff
@@ -0,0 +1,54 @@
+# identifierCase
+
+Converts identifiers to upper or lowercase.
```

Comment:
This might be a good place to specify exactly what classifies as an identifier and what doesn't.

Reply:
Okay with this text?

Reply:
I added a note in commit [afe5b48](https://github.com/sql-formatter-org/sql-formatter/pull/663/commits/afe5b48b281957e5904b9d6e271d67c1f272e595) - is it okay?